File: src/org/baksia/rustycage/editors/RustTextHover.java
package org.baksia.rustycage.editors;

import org.eclipse.jface.text.IRegion;
import org.eclipse.jface.text.ITextHover;
import org.eclipse.jface.text.ITextViewer;

public class RustTextHover implements ITextHover {

	@Override
	public String getHoverInfo(ITextViewer textViewer, IRegion hoverRegion) {
		
		return null;
	}

	@Override
	public IRegion getHoverRegion(ITextViewer textViewer, int offset) {
		return null;
	}

}

File: rustycage-plugin/src/org/rustycage/editors/RustTextHover.scala
package org.rustycage.editors

import org.eclipse.jface.text.ITextHover
import org.eclipse.jface.text.ITextViewer
import org.eclipse.jface.text.IRegion

class RustTextHover extends ITextHover {
	def getHoverInfo(textViewer: ITextViewer , hoverRegion: IRegion ): String = {
	  null
	}
	
	def getHoverRegion(textViewer: ITextViewer , offset: Int ): IRegion = {
	  null
	}
}